examples/step-89/step-89.cc

```diff
@@ -1721,7 +1721,7 @@ int main(int argc, char *argv[])
 
   pcout << "Create DoFHandler..." << std::endl;
   DoFHandler<dim> dof_handler(tria);
-  dof_handler.distribute_dofs(FESystem<dim>(FE_DGQ<dim>(degree), dim + 1));
+  dof_handler.distribute_dofs(FESystem<dim>(FE_DGQ<dim>(degree) ^ (dim + 1)));
```

Comment:
This would not work on Intel 18 compilers, as pointed out [here](https://dealii.org/developer/doxygen/deal.II/classFESystem.html#a1572f6043dc6ed97fd3b541eff104827).

Do we still care about Intel 18?

Reply:
We already use it in 17,18,22,29,31,32,etc.

Reply:
I don't recall the PR number, but we had that change queued for a couple of releases and merged it after the last one.